src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -671,7 +671,7 @@ private function fixPhpDocType(string $type, array $uses, string $namespaceName)
         $typeExpression = $typeExpression->mapTypes(function (TypeExpression $type) use ($uses, $namespaceName) {
             $currentTypeValue = $type->toString();
 
-            if ($type->isCompositeType() || !Preg::match('/^'.self::REGEX_CLASS.'$/', $currentTypeValue)) {
+            if ($type->isCompositeType() || !Preg::match('/^'.self::REGEX_CLASS.'$/', $currentTypeValue) || \in_array($currentTypeValue, ['min', 'max'], true)) {
```

Comment:
NOTE: I belive the issue is about `min` and `max` solely, as other "types" like `positive-int` always contain `-` (dash) which is never a valid class name, thus never fixed.

Reply:
I've checked for some others in PHPStan's codebase, especially in tests, but there are none.